sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/AzureBlobFSLinkedService.cs

```diff
@@ -80,7 +80,12 @@ public AzureBlobFSLinkedService()
         /// AzureKeyVaultSecretReference. If servicePrincipalCredentialType is
         /// 'ServicePrincipalCert', servicePrincipalCredential can only be
         /// AzureKeyVaultSecretReference.</param>
-        public AzureBlobFSLinkedService(object url, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), IntegrationRuntimeReference connectVia = default(IntegrationRuntimeReference), string description = default(string), IDictionary<string, ParameterSpecification> parameters = default(IDictionary<string, ParameterSpecification>), IList<object> annotations = default(IList<object>), object accountKey = default(object), object servicePrincipalId = default(object), SecretBase servicePrincipalKey = default(SecretBase), object tenant = default(object), object azureCloudType = default(object), object encryptedCredential = default(object), CredentialReference credential = default(CredentialReference), object servicePrincipalCredentialType = default(object), SecretBase servicePrincipalCredential = default(SecretBase))
+        /// <param name="sasUri">SAS URI of the Azure Data Lake Storage Gen2
+        /// service. Type: string, SecureString or
+        /// AzureKeyVaultSecretReference.</param>
+        /// <param name="sasToken">The Azure key vault secret reference of
+        /// sasToken in sas uri.</param>
+        public AzureBlobFSLinkedService(IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), IntegrationRuntimeReference connectVia = default(IntegrationRuntimeReference), string description = default(string), IDictionary<string, ParameterSpecification> parameters = default(IDictionary<string, ParameterSpecification>), IList<object> annotations = default(IList<object>), object url = default(object), object accountKey = default(object), object servicePrincipalId = default(object), SecretBase servicePrincipalKey = default(SecretBase), object tenant = default(object), object azureCloudType = default(object), object encryptedCredential = default(object), CredentialReference credential = default(CredentialReference), object servicePrincipalCredentialType = default(object), SecretBase servicePrincipalCredential = default(SecretBase), object sasUri = default(object), SecretBase sasToken = default(SecretBase))
```

Comment:
The two new parameters are optional parameters added in the end. I think this is a backward compatible change. You don't need to add back the original constructor in custom code.

Reply:
We also removed url from requried parameters. So the first parameter is not url in further.